src/jose/jws/general/verifier.ts

```diff
@@ -0,0 +1,112 @@
+import type { GeneralJws, Signature } from './types';
+import type { JwkPublic, VerifyFn } from '../../types';
+import type { VerificationMethod } from '../../../did/did-resolver';
+
+import { base64url } from 'multiformats/bases/base64';
+import { DIDResolver } from '../../../did/did-resolver';
+import { verify as verifyEd25519 } from '../../algorithms/ed25519';
+import { verify as verifySecp256k1 } from '../../algorithms/secp256k1';
+
+const verifiers: { [key:string]: VerifyFn } = {
+  'Ed25519'   : verifyEd25519,
+  'secp256k1' : verifySecp256k1
+};
+
+type VerificationResult = {
+  signers: string[];
```

Comment:
Not clear what the result contain. Perhaps:
```suggestion
  /**
   * DID of list of signers that passed signature verification.
   */
  verifiedSigners: string[];
```

Reply:
https://github.com/TBD54566975/dwn-sdk-js/commit/f9ab6086604c1499eb8cc699d9d492ba0a705113

Reply:
`verifiedSigners` feels a bit redundant given that the method throws an exception if any signature verification fails.

Reply:
Oh... I must've read the code wrong, I thought the code would continue if any signature verification failed. Will read the code again!